src/ert/scheduler/openpbs_driver.py

```diff
@@ -66,6 +81,51 @@ def _resource_string(self) -> str:
             resource_specifiers += ["mem=" + self._memory_per_job]
         return ":".join(resource_specifiers)
 
+    async def _execute_with_retry(
+        self,
+        cmd_with_args: List[str],
+        exit_codes_triggering_retries: List[int],
+    ) -> Tuple[bool, str]:
+        error_message: Optional[str] = None
+
+        for _ in range(self._num_retries):
+            try:
+                process = await asyncio.create_subprocess_exec(
+                    *cmd_with_args,
+                    stdout=asyncio.subprocess.PIPE,
+                    stderr=asyncio.subprocess.PIPE,
+                )
+                stdout, stderr = await process.communicate()
+
+                if process.returncode == 0:
+                    return True, stdout.decode().strip()
+                elif process.returncode in exit_codes_triggering_retries:
+                    # expected exit code when failed command
```

Comment:
don't see what extra information this comment gives.

Reply:
do you want to assert stderr?

Reply:
Sorry, left over. Delete ....